data/members.csv

```diff
@@ -0,0 +1,19 @@
+Firstname,Lastname,Affiliation,SWE_WG_Member,MMRM
+Alessandro,Gasparini,Karolinska Institutet,1,
```

Comment:
I guess it would be a bit better to have explicit `0` e.g. for the `no` fields, e.g. here
```suggestion
Alessandro,Gasparini,Karolinska Institutet,1,0
```